spec/package.spec.json

```diff
@@ -161,11 +161,12 @@
       "description": "Data for a deployed instance of a contract",
       "type": "object",
       "required": [
-        "contract_type"
+        "contract_type",
+        "address"
```

Comment:
Nice catch!

Reply:
It's all TypeScript saving my bacon :bacon: